example.php

```diff
@@ -48,436 +112,257 @@ function getSSLPage($url) {
-    $language->setExecutableName('appwrite');
-    $language->setLogo(json_encode("
-    _                            _ _           ___   __   _____ 
+    if (!$requestedSdk || $requestedSdk === 'cli') {
+        $language  = new CLI();
+        $language->setNPMPackage('appwrite-cli');
+        $language->setExecutableName('appwrite');
+        $language->setLogo(json_encode("
+    _                            _ _           ___   __   _____
    /_\  _ __  _ ____      ___ __(_) |_ ___    / __\ / /   \_   \
   //_\\\| '_ \| '_ \ \ /\ / / '__| | __/ _ \  / /   / /     / /\/
- /  _  \ |_) | |_) \ V  V /| |  | | ||  __/ / /___/ /___/\/ /_  
- \_/ \_/ .__/| .__/ \_/\_/ |_|  |_|\__\___| \____/\____/\____/  
-       |_|   |_|                                                
+ /  _  \ |_) | |_) \ V  V /| |  | | ||  __/ / /___/ /___/\/ /_
+ \_/ \_/ .__/| .__/ \_/\_/ |_|  |_|\__\___| \____/\____/\____/
+       |_|   |_|
 
 "));
-    $language->setLogoUnescaped("
-     _                            _ _           ___   __   _____ 
+        $language->setLogoUnescaped("
+     _                            _ _           ___   __   _____
     /_\  _ __  _ ____      ___ __(_) |_ ___    / __\ / /   \_   \
    //_\\\| '_ \| '_ \ \ /\ / / '__| | __/ _ \  / /   / /     / /\/
-  /  _  \ |_) | |_) \ V  V /| |  | | ||  __/ / /___/ /___/\/ /_  
-  \_/ \_/ .__/| .__/ \_/\_/ |_|  |_|\__\___| \____/\____/\____/  
+  /  _  \ |_) | |_) \ V  V /| |  | | ||  __/ / /___/ /___/\/ /_
+  \_/ \_/ .__/| .__/ \_/\_/ |_|  |_|\__\___| \____/\____/\____/
         |_|   |_|                                                ");
 
-    $sdk  = new SDK($language, new Swagger2($spec));
-
-    $sdk
-        ->setName('NAME')
-        ->setVersion('0.16.0')
-        ->setDescription('Repo description goes here')
-        ->setShortDescription('Repo short description goes here')
-        ->setURL('https://appwrite.io')
-        ->setLogo('https://appwrite.io/v1/images/console.png')
-        ->setLicense('BSD-3-Clause')
-        ->setLicenseContent('test test test')
-        ->setWarning('**WORK IN PROGRESS - NOT READY FOR USAGE**')
-        ->setChangelog('**CHANGELOG**')
-        ->setGitUserName('appwrite')
-        ->setGitRepoName('sdk-for-cli')
-        ->setTwitter('appwrite_io')
-        ->setDiscord('564160730845151244', 'https://appwrite.io/discord')
-        ->setDefaultHeaders([
-            'X-Appwrite-Response-Format' => '1.7.0',
-        ])
-        ->setExclude([
-            'services' => [
-                ['name' => 'assistant'],
-                ['name' => 'avatars'],
+        $sdk  = new SDK($language, new Swagger2($spec));
+
+        configureSDK($sdk, [
+            'version' => '0.16.0',
+            'url' => 'https://appwrite.io',
+            'license' => 'BSD-3-Clause',
+            'gitUserName' => 'appwrite',
+            'gitRepoName' => 'sdk-for-cli',
+            'defaultHeaders' => ['X-Appwrite-Response-Format' => '1.7.0'],
```

Comment:
some have 1.6.0. can be a static value as well. we can try to keep things as same as possible if this is only for local tests.